pkcs7/Cargo.toml

```diff
@@ -15,15 +15,29 @@ edition = "2021"
 rust-version = "1.65"
 
 [dependencies]
-der = { version = "0.7", features = ["oid"] }
+const-oid = { version = "0.9", features = ["db"] }
+der = { version = "0.7", features = ["oid", "std"] } # TODO NM "std" for builder only
 spki = { version = "0.7" }
 x509-cert = { version = "0.2", default-features = false }
 
+# optional dependencies
+sha1 = { version = "0.10"}
+sha2 = { version = "0.10"}
+sha3 = { version = "0.10"}
+signature = { version = "2.1.0", features = ["digest", "alloc"]}
+
 [dev-dependencies]
 der = { version = "0.7", features = ["oid", "pem"] }
+rsa = { version = "0.9.0-pre.1", features = ["sha2"] }
```

Comment:
0.9.0 final is out.

Reply:
(and I would recommend the 0.9.1, it has a fix for signature padding)

Reply:
Updated.